net-analyzer/nuttcp/files/nuttcp-8.1.4-makefile.patch

```diff
@@ -0,0 +1,21 @@
+--- a/Makefile	2017-10-29 08:39:34.209474792 +0200
++++ b/Makefile	2017-10-29 15:11:02.082490459 +0200
+@@ -2,8 +2,7 @@
+ #EXTRAVERSION=-pre1
+ CC = gcc -Wall
+ #OPT = -g -O0
+-OPT = -O3
+-CFLAGS = $(OPT) $(NOIPV6)
++CFLAGS += $(OPT) $(NOIPV6)
```

Comment:
Why are you adding OPT if you just removed it? ;-)

If only the one make build rule edited below is needed, I think you can get the same result without patching anything, via running `emake OPT="${CFLAGS} ${CPPFLAGS} ${LDFLAGS}"`.

Reply:
I removed OPT defined with -O3.
"CFLAGS = $(OPT) $(NOIPV6)" - this removed because it will ignore CFLAGS from make.conf